playwright-e2e/utils/api-utils.ts

```diff
@@ -70,6 +75,7 @@ export async function getAuth0AccessToken(app: APP): Promise<string> {
   }
 
   const credentials = JSON.parse(buildAuth0ClientCredentials(app));
+  console.log(`credentials: ${JSON.stringify(credentials)}`);
```

Comment:
Is this for future debugging, or just something needed for development of the test?

Reply:
I meant to change those console.logs into logInfo for future debugging - will make sure to make the change, thanks for catching!